Marlin/src/libs/MAX31865.cpp

```diff
@@ -81,6 +81,7 @@ SPISettings MAX31865::spiConfig = SPISettings(
     _mosi = spi_mosi;
     _miso = spi_miso;
     _sclk = spi_clk;
+    _spi_speed = swSpiInit(SPI_QUARTER_SPEED, SD_SCK_PIN, SD_MOSI_PIN);
```

Comment:
I think this should use `spi_mosi`, `spi_sck` since the SD Card pins will not always be used, so one could use arbitrary free pins for temp sensors. In this case, those are provided in the constructor. It looks like the actual/provided pins are used elsewhere; unsure if this is just a requirement for the setup routine or not.

Also, this disregards the speed selected in spiConfig above; can `_spi_speed` be integrated into or replace `spiConfig` above, as well as the `SPI_QUARTER_SPEED`?

Reply:
Let me change this. This is definitely a bug. I copied and pasted the code from elsewhere and i forgot to change the pins

Reply:
>Also, this disregards the speed selected in spiConfig above; can _spi_speed be integrated into or replace spiConfig above, as well as the SPI_QUARTER_SPEED?

To be honest I am not too sure myself. From what I can tell, spiConfig has no accessible fields or functions to allow me to read the configuration that was passed in. I would assume the next best thing is to simply declare a separate private field which can be shared. What are your thoughts?